index.html

```diff
@@ -94,103 +99,107 @@ <h2>Multi-Post Stories</h2>
+                    <li><a href=""><img src="images/portfolio1.svg" alt=""></a></li>
+                    <li><a href=""><img src="images/Linkedin icon.svg" alt=""></a></li>
+                    <li><a href=""><img src="images/portfolio4.svg" alt=""></a></li>
+                    <li><a href=""><img src="images/portfolio3.svg" alt=""></a></li>
+                    <li><a href=""><img src="images/portfolio5.svg" alt=""></a></li>
+                </ul>
+                <button class="btn-action" id="about-btn">Get my resume</button>
+            </div>
+            <div class="skill-container">
+                <div class="languages">
+                    <h2>Languages</h2>
+                    <i class="fa fa-chevron-down"></i>
+                </div>
+                <ul class="language-list">
+                    <li>
+                        <img src="images/Frame 9.png" alt="Javascript">
+                    </li>
+                    <li>
+                        <img src="images/Frame 10.png" alt="HTML">
+                    </li>
+                    <li>
+                        <img src="images/Frame 11.png" alt="CSS">
+                </ul>
+                <ul class="frames">
+                    <li>
+                        <span>
+                            <h2>Frameworks</h2><i class="fa fa-chevron-right"></i>
+                        </span>
+                    </li>
+                    <hr />
+                    <li>
+                        <span>
+                            <h2>Skills</h2><i class="fa fa-chevron-right"></i>
+                        </span>
+                    </li>
+                    <hr />
+                </ul>
             </div>
-            <ul class="language-list">
-                <li>
-                    <img src="images/Ellipse 1.png" alt="Javascript">
-                    <h2>Javascript</h>
-                </li>
-                <li>
-                    <img src="images/Ellipse 1 (2).png" alt="HTML">
-                    <h2>HTML</h2>
-                </li>
-                <li>
-                    <img src="images/Ellipse 1 (3).png" alt="CSS">
-                    <h2>CSS</h2></li>
-            </ul>
-            <ul class="frames">
-                <li>
-                    <span><h2>Frameworks</h2><i class="fa fa-chevron-right"></i></span>
-                </li>
-                <hr/>
-                <li>
-                    <span><h2>Skills</h2><i class="fa fa-chevron-right"></i></span>
-                </li>
-                <hr/>
-            </ul>
-           </div>
         </div>
         <div class="footer_section">
             <div class="footer_container">
                 <h1 class="footer_header">Contact me</h1>
                 <p>If you have an apllication you
-                    are interested in developing, a 
-                    feature that you need built or 
+                    are interested in developing, a
+                    feature that you need built or
```

Comment:
Figma
![image](https://github.com/Eugeneiyukhun/Setup_and_Mobile_First/assets/78251927/6e935d13-51dc-492d-a805-00d310ddfe48)

Your design
![image](https://github.com/Eugeneiyukhun/Setup_and_Mobile_First/assets/78251927/ac2333b0-2e66-4fb2-92d6-a447eedb0136)

![image](https://github.com/Eugeneiyukhun/Setup_and_Mobile_First/assets/78251927/9fa224e7-43aa-4666-a960-cfe4a9eca465)

Please let's adjust our design as much as possible using [the info of the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

- Section background images do not tally
- Let's place the content of the section in the center.
- It would be nice if we reduce the width of the contact form according to Figma
- Kindly remove the excess white space at the bottom of your design. it is not present in the original design 👍